libraries/AP_ICEngine/AP_ICEngine.h

```diff
@@ -107,6 +107,10 @@ class AP_ICEngine {
     // delay between start attempts (seconds)
     AP_Float starter_delay;
 
+    // max crank retry
+    AP_Int8 max_crank_retry;
+    int8_t crank_retry_ct;
```

Comment:
Why not a uint8_t?

Reply:
Basically just felt bad mixing signed and unsigned for the two, and there's no AP_Uint8 for the param.